src/lang/modifyAst.ts

```diff
@@ -62,10 +53,18 @@ import { KCL_DEFAULT_CONSTANT_PREFIXES } from '@src/lib/constants'
 import type { DefaultPlaneStr } from '@src/lib/planes'
 
 import { ARG_AT } from '@src/lang/constants'
-import type { Coords2d } from '@src/lang/std/sketch'
+import { isLiteralArrayOrStatic, type Coords2d } from '@src/lang/util'
 import { err, trap } from '@src/lib/trap'
 import { isArray, isOverlap, roundOff } from '@src/lib/utils'
 import type { ExtrudeFacePlane } from '@src/machines/modelingSharedTypes'
+import {
+  type addTagForSketchOnFace as AddTagForSketchOnFaceFn,
+  type getConstraintInfoKw as getConstraintInfoKwFn,
+} from '@src/lang/std/sketch'
+import {
+  type removeSingleConstraint as removeSingleConstraintFn,
+  type transformAstSketchLines as transformAstSketchLinesFn,
```

Comment:
You can see how the TS LSP picks up on `AddTagForSketchOnFaceFn` as a type by highlighting it with a different color from `getConstraintInfoKwFn` etc

```suggestion
  type getConstraintInfoKw as GetConstraintInfoKwFn,
} from '@src/lang/std/sketch'
import {
  type removeSingleConstraint as RemoveSingleConstraintFn,
  type transformAstSketchLines as TransformAstSketchLinesFn,
```

Reply:
Nit though